sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2400,6 +2401,25 @@ object functions {
   def weekofyear(columnName: String): Column = weekofyear(Column(columnName))
 
   //////////////////////////////////////////////////////////////////////////////////////////////
+  // Collection functions
+  //////////////////////////////////////////////////////////////////////////////////////////////
+
+  /**
+   * Returns length of array or map
+   * @group collection_funcs
+   * @since 1.5.0
+   */
+  def size(columnName: String): Column = size(Column(columnName))
```

Comment:
Nit: We just removed the functions with the column name, probably we can leave it for the further updating.